docs/When_Do_You_Need_A_Custom_Device.md

```diff
@@ -0,0 +1,73 @@
+#### When do you Need a Custom Device?
+
+The built-in components of an VeriStand Project are listed in **[VeriStand Help](https://www.ni.com/documentation/en/veristand/latest/manual/manual-overview/)** » **[VeriStand Environment](https://www.ni.com/documentation/en/veristand/latest/manual/environment/)** » **System Explorer** section in the table. If the built-in components do not fulfill a specification, it can most likely be fulfilled by one of the customization methods shown in NI Developer Zone Tutorial: Using LabVIEW and Other Software Environments with VeriStand.
+
+
+Although VeriStand provides most of the general functionality required by a real-time testing application, NI has designed the environment so that you can customize and extend it when necessary to ensure it always meets your application requirements. A collection of [NI supported VeriStand add-ons as well as additional add-ons](https://www.ni.com/en-us/support/documentation/supplemental/15/veristand-add-ons.html) gathered from internal NI developers and the VeriStand community has been developed and these custom devices may fulfill your specifications.
```

Comment:
```suggestion
Although VeriStand provides most of the general functionality required by a real-time testing application, NI has designed the environment so that you can customize and extend it when necessary to ensure it always meets your application requirements. A collection of [NI supported VeriStand add-ons as well as additional add-ons](https://www.ni.com/en-us/support/documentation/supplemental/15/veristand-add-ons.html) gathered from internal NI developers and the VeriStand community has been developed and these custom devices may fulfill your specifications.

```